cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/ModelUtils.java

```diff
@@ -101,7 +101,7 @@ public static Double estimateLeaderCpuUtilPerCore(double brokerCpuUtil,
     if (useLinearRegressionModel) {
       return estimateLeaderCpuUtilUsingLinearRegressionModel(partitionBytesInRate, partitionBytesOutRate);
     } else {
-      if (brokerLeaderBytesInRate == 0 || brokerLeaderBytesOutRate == 0) {
+      if (brokerLeaderBytesInRate == 0 && brokerLeaderBytesOutRate == 0) {
```

Comment:
I think we can also add comments in the javadoc of this method, to clarify this bytesOut does not include replication traffic.

Reply:
Will do, thanks